pubspec.yaml

```diff
@@ -69,6 +69,8 @@ dependencies:
   tuple: ^2.0.2
   collection: 1.17.1
   material_color_utilities: ^0.2.0
+  dots_indicator: ^2.1.1
+  flutter_markdown: ^0.6.18
```

Comment:
https://github.com/search?q=repo%3Aflutter%2Fflutter+flutter_markdown&type=issues&p=2

Some issues to track on this package. I see it's a Flutter-built package, so there should be no issues using it, but when we get into less obvious functionality like a11y/screen readers, zooming/text resizing, etc. we need to double check that the markdown content works properly